### ISSUE TITLE
German Translation updated for r243 - only conversationlist is missing

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#303]
```
Updated all the German Language files for r243.
The upcoming conversationlist is not yet included. (Has to be merged first)
```

Original issue reported on code.google.com by `SamuelPl...@gmail.com` on 12 May 2012 at 8:37

Attachments:
- [Translation.zip](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-303/comment-0/Translation.zip)

**Comment by GoogleCodeExporter** [comment]
Original comment by `sdeva...@gmail.com` on 14 May 2012 at 1:05
- Added labels: **Type-Translation**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Updated all the German Language files for r251.
The conversationlist is only partly included. (Has to be reviewed from unzel on)
```

Original comment by `SamuelPl...@gmail.com` on 29 May 2012 at 12:57

Attachments:
- [Patch.zip](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-303/comment-2/Patch.zip)

**Comment by GoogleCodeExporter** [comment]
```
Thanks, looks good.

This is now committed in r252.
```

Original comment by `oskar.wi...@gmail.com` on 30 May 2012 at 6:59
- Changed state: **Ready**
- Added labels: **Milestone-0.6.11**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
